department-of-veterans-affairs/caseflow
# Create FNOD Banner component

Resolves [CASEFLOW-241](https://vajira.max.gov/browse/CASEFLOW-241)

### Description
Create FNOD Banner indicating date of death has been reported

### Acceptance Criteria
- [ ] Banner renders if there is a date of death for a Veteran appellant

### Testing Plan
1. In the `veterans` table, add a date of death string e.g. `2019-03-17` to a veteran file number like 500000000
3. Go to http://localhost:3000/queue
4. Search for the same veteran file number 
5. Verify that the FNOD badge appears in the Case List Table results
6. Click on the file number
7. Verify the badge appears on the Case Details page
8. Verify that the banner appears below the Case Title 

### User Facing Changes

 BEFORE|AFTER
 ---|---
<img width="1123" alt="Screen Shot 2021-01-04 at 7 08 48 PM" src="https://user-images.githubusercontent.com/68879427/103603022-5d3f8d00-4ec2-11eb-9908-af6c90883593.png">|<img width="1129" alt="Screen Shot 2021-01-06 at 9 21 38 AM" src="https://user-images.githubusercontent.com/68879427/103799964-a3026f80-5000-11eb-9081-671455945e5c.png">

### Storybook Story
- [ ] Add a [Storybook](https://github.com/department-of-veterans-affairs/caseflow/wiki/Documenting-React-Components-with-Storybook) file alongside the component file
